pandas/core/frame.py

```diff
@@ -2133,6 +2133,8 @@ def to_gbq(
         """
         Write a DataFrame to a Google BigQuery table.
 
+        .. deprecated:: 2.2.0
+
```

Comment:
We should add a message telling users to use ``pandas-gbq`` directly instead.

Reply:
i.e. ``pandas_gbq.read_gbq``

Reply:
Good idea. Added